pluto_build/bin/setup.sh

```diff
@@ -1,12 +1,17 @@
 #!/bin/bash
 
 function setup {
-sudo apt update
-sudo apt install -y curl zip
+    sudo apt update
+    sudo apt install -y curl zip
+    sudo apt install python3-pip
+    pip3 install csv2md
+    sudo apt-get update
 
-curl -O https://dl.min.io/client/mc/release/linux-amd64/mc
-chmod +x mc
-sudo mv ./mc /usr/bin
-mc config host add spaces $AWS_S3_ENDPOINT $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY --api S3v4
+    curl -O https://dl.min.io/client/mc/release/linux-amd64/mc
+    chmod +x mc
+    sudo mv ./mc /usr/bin
+    mc config host add spaces $AWS_S3_ENDPOINT $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY --api S3v4
+
+    docker-compose up -d
```

Comment:
Does this line mean that all subsequent bash commands will be run through the ubuntu container? FacDB is more explicit, using `docker-compose exec <commands>` to run each step via the containers. I figured pluto would have similar syntax as we switch to running on a container even on github actions, but I haven't used docker compose enough to know for sure what commands we need to achieve this

Reply:
Ok cool. My take here is this is an enormously important line, as it changes the OS on which the entire pipeline is run. It's an easy thing for a engineer who less unfamiliar with docker compose (like me) to miss and then get very confused going forward. I think [facdb's implementation](https://github.com/NYCPlanning/db-facilities/blob/86729a33c528bcdcb83ebbbc3c98a16b6d3bc9ef/facdb.sh#L7)
```
docker-compose exec -T facdb facdb init
```
is easier to understand. I'm not saying we should add in those kinds of statements, but maybe there is a way to make our implementation more explicit for the reader?

Reply:
And how do know that it is working? Is there an action run off the 332 branch that successfully runs in a container?